Makefile

```diff
@@ -12,6 +12,8 @@ endif
 
 DOCKER_COMPOSE = docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.${TARGET}.yml
 
+EXIT = exit=$$?; $(MAKE) stop; exit $$exit
```

Comment:
Name suggestions?

Reply:
`STOP`?

Reply:
If this wasn't inserted in a command, the original exit code will be used. This behavior is preserved and an additional stop is inserted, so I think `STOP` captures the intention.